corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -270,6 +271,17 @@ def _print_table(self, rows, output):
             ["Index CName", "Index Name", "Size on Disk", "Doc Count"], rows=rows
         )
 
+    def set_replicas_for_secondary_index(self, cname):
+        adapter = doc_adapter_from_cname(cname)
+
+        if not getattr(settings, f'ES_{cname.upper()}_INDEX_MULTIPLEXED'):
+            raise IndexNotMultiplexedException("""This command supports setting replicas
+                                               only in secondary index of multiplexed Indices.""")
+        if getattr(settings, f'ES_{cname.upper()}_INDEX_SWAPPED'):
+            raise IndexAlreadySwappedException("Replicas can only be set before swapping indexes.")
```

Comment:
What happens if this step is forgotten until after the indexes are swapped? Is it necessary/possible to un-swap them, and then run this command, and finally swap them again? If that works, why not allow it to run on a swapped index? As in
```py
self._prepare_index_for_normal_usage(adapter.primary if swapped else adapter.secondary)
```

Reply:
I would want it to be run before swapping the indices because running a cluster with only primary shards would not be a good idea. 
Anyways this is just for internal use and would be used for indices where we are facing issues while reindexing and I don't think we would be exposing it in our external changelogs.

Reply:
And if required we can alway set it from django-manage shell using the `ElasticManageAdapter`